flutter/pubspec.yaml

```diff
@@ -29,17 +29,17 @@ dev_dependencies:
   flutter_lints: ^3.0.1
   flutter_test:
     sdk: flutter
-  freezed: ^2.2.0
+  freezed: 2.5.3
   gherkin: ^3.1.0
   import_sorter: ^4.6.0
   json_serializable: ^6.7.1
 
 # TODO: Remove once `flutter_gherkin` updates.
 dependency_overrides:
-  analyzer: ^5.13.0
-  # Pinned until analyzer is upgraded to v6
-  #  https://github.com/dart-lang/dart_style/issues/1328
-  dart_style: 2.3.3
+  # Pinned until `flutter_gherkin` updates
+  analyzer: ^6.5.0
+  # Pinned until `flutter_gherkin` updates
+  meta: ^1.14.0
   # Pinned until gql-dart updates to v4
   # https://github.com/gql-dart/gql/blob/master/links/gql_websocket_link/pubspec.yaml#L13
```

Comment:
@evdokimovs, I guess this can be removed, since `gql-dart` supports uuid v4 now.

Reply:
@SleepySquash ,

`gql-dart` is not a problem here now, but `flutter_gherkin` is still requires `uuid v3.0.7`.

> `flutter_gherkin`
> The latest prerelease was [3.0.0-rc.17](https://pub.dev/packages/flutter_gherkin/versions#prerelease) [21 months ago](https://pub.dev/packages/flutter_gherkin/versions).

I think that it will never be updated... Will consider migration to something else in the future.